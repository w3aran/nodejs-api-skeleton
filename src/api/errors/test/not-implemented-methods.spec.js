const { notImplementedMethods } = require("../not-implemented-methods");

describe("notImplementedMethods", () => {
  it("should return hapi route object", () => {
    expect(notImplementedMethods("/")).toMatchSnapshot();
  });
});
