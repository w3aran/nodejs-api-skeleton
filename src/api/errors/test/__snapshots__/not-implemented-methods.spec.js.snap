// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notImplementedMethods should return hapi route object 1`] = `
Object {
  "handler": [Function],
  "method": "*",
  "path": "/",
}
`;
